src/FSharp.Stats/Matrix.fs

```diff
@@ -15,220 +15,223 @@ open System.Diagnostics
 //type vector = Vector<float>
 //type rowvec = RowVector<float>
 
-module MRandom = 
+module MRandom =
     let randomGen = new System.Random()
-    let float f = randomGen.NextDouble() * f 
-
+    let float f = randomGen.NextDouble() * f
 
 [<CompilationRepresentation(CompilationRepresentationFlags.ModuleSuffix)>]
 module Matrix = begin
-        
+
     module Generic = begin
 
         module MS = SpecializedGenericImpl
 
         // Accessors
-        let get (a:Matrix<_>) i j   = a.[i,j]
-        let set (a:Matrix<_>) i j x = a.[i,j] <- x              
+        let get (matrix:Matrix<_>) index1 index2 = matrix.[index1,index2]
```

Comment:
change index1 and index2 to indexCol and indexRow

Reply:
Changed variables index_ and length_ to _Row and _Col